tests/fixtures_server.py

```diff
@@ -123,51 +123,69 @@ def get_patched_settings(temp_path: Path):
     return settings
 
 
+modules_to_patch = [
```

Comment:
Maybe we can capitalize it as in `MODULES_TO_PATCH`? This is sometimes our (informal) way of denoting variables which are harcoded or computed in the main block of a module (i.e. outside functions/classes).